Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -416,16 +416,18 @@ export async function startAppService(endpoint, resourceGroupName: string, webAp
         'Authorization': 'Bearer '+ accessToken
     };
     var webAppNameWithSlot = (specifySlotFlag) ? webAppName + '-' + slotName : webAppName;
+    tl.debug('Request to start App Service: ' + url);
```

Comment:
What are these changes in start app service? They don't seem related to the extension changes.